CAMP.md

```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
 
 The following endpoints are required:
 
-## POST /v1/users
+## POST /v1/domains/{domain}/users
 
 No Authorization (Public method)
 
 Creates a new user. The new user is associated with the host's domain by default. This only changes if the endpoint described in the ([CAPP](/CAPP.md)) sub-protocol is used. The user<->domain relationship is one-to-one, although one server can have users with the same name as long as they are on different domains.
 
-Body:
-```javascript
+Request:
+```
+POST /domains/{domain}/users HTTP/1.1
+
 {
-    "username": "john", // the username that will be the first part of the alias
+    "username": "alice",
     "password": "mysecretpassword"
 }
 ```
 
 Response:
-```javascript
-Status: 200
-{}
 ```
+HTTP/1.1 200 OK
 
-## POST /v1/auth
+```
 
-Authorization Bearer {jwt}
+

```

Comment:
Authorization is traditionally done via a POST. I think it makes more sense the way it was before with going to v1/auth and sending the data as json. Again, this is just more industry standard for authentication. your JWT example is better though see we can keep that

Reply:
Another reason why I don't like this is the path is inconsistent, the rest of our endpoints are /domains/users/assets this one randomly changes to /domains/users/tokens